source/Body.cpp

```diff
@@ -20,6 +20,7 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include "GameData.h"
 #include "Mask.h"
 #include "MaskManager.h"
+#include "Outfit.h"
```

Comment:
Where is this used?

Reply:
Good catch, I had initially used this to check whether certain sprites should be triggered if some outfits were installed but now I use the ConditionStore. Will remove this include!